fish-rust/src/env/environment_impl.rs

```diff
@@ -339,57 +339,61 @@ impl EnvScopedImpl {
         self.perproc_data.statuses = s;
     }
 
-    #[widestrs]
     fn try_get_computed(&self, key: &wstr) -> Option<EnvVar> {
         let ev = ElectricVar::for_name(key);
         if ev.is_none() || !ev.unwrap().computed() {
             return None;
         }
 
-        if key == "PWD"L {
+        if key == L!("PWD") {
```

Comment:
these comparisons should work fine with plain string literals, which is preferrable. Same elsewhere.

Reply:
Done, except for one - https://github.com/fish-shell/fish-shell/blob/ca972f6e0fbc3dc236e68b9ebfcbe2deb4a48be1/src/future_feature_flags.rs#L189

This gives "^^ no implementation for `[char] == str`". So that one gets a `L!()`.